src/winston-logger.ts

```diff
@@ -64,23 +65,38 @@ export const createLogger = (loggerConfiguration: LoggerConfiguration) => {
     // tslint:disable:no-console
     logger.on('error', error => console.error('logger Error!', error));
 
-    if (loggerConfiguration.logstashConfigurations) {
-        loggerConfiguration.logstashConfigurations.forEach(logstashConfiguration => {
-            const logstashTransport = new LogstashTransport({
-                host: logstashConfiguration.logstashHost,
-                port: logstashConfiguration.logstashPort,
+    if (loggerConfiguration.udpConfigurations) {
+        loggerConfiguration.udpConfigurations.forEach((udpConfiguration: SocketAddress) => {
```

Comment:
For each logstash config, check the protocol and create the appropriate tranport

Reply:
OK, do we want to make the protocol optional in the logstashConfiguration? If so then we will need to make a default protocol, what will it be? 

On the other hand we can make it required, but it will be a breaking change.
What do you propose?

Reply:
breaking change, and required